tests/test_questions.py

```diff
@@ -676,6 +715,42 @@ def test_is_empty_optional_question(self):
         assert question.is_empty
 
 
+class TestDateSummary(QuestionSummaryTest):
+
+    def question(self, **kwargs):
+        data = {
+            "id": "example",
+            "type": "date"
+        }
+        data.update(kwargs)
+
+        return ContentQuestion(data)
+
+    def test_date_is_formatted_into_user_friendly_format(self):
+        question = self.question().summary({'example': '2016-02-18'})
```

Comment:
Is it worth having a test for where we don't include 0 padding i.e. `2`?

Either we are saving the month input directly in which case we may see data like `{'example': '2017-2-19'}` in our DB, and therefore we need to know our DateSummary is able to handle that or we only save data that includes 0 padding.